modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/store/GroupPageStoreHolderMap.java

```diff
@@ -17,20 +17,21 @@
 
 package org.apache.ignite.internal.pagememory.persistence.store;
 
+import java.util.List;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.function.BiFunction;
 import java.util.function.Function;
 
 /**
- * Proxy class for map (grpId -> page store holder) that wraps data adding and replacing operations to disallow concurrent execution
+ * Proxy class for map (grpId -> page store list) that wraps data adding and replacing operations to disallow concurrent execution
  * simultaneously with cleanup of file page storage.
  *
  * <p>Wrapping of data removing operations is not needed.
  *
  * @param <T> Type of {@link PageStore}.
  */
-class GroupPageStoreHolderMap<T extends PageStore> extends ConcurrentHashMap<Integer, GroupPageStoreHolder<T>> {
+class GroupPageStoreHolderMap<T extends PageStore> extends ConcurrentHashMap<Integer, List<T>> {
```

Comment:
This class should be renamed

Reply:
What about **GroupPageStoreHolderMap** -> **GroupPageStoresMap** ?

Reply:
Made a composition.

Reply:
Discussed personally, we apply my proposals for renaming.